### PR TITLE
feat: update permission for ebs backup and restore

### PR DESCRIPTION
**Pull request by fengou1**
- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### What is changed, added, or deleted? (Required)

Update backup requires permission for backup and restore

### Which TiDB Operator version(s) do your changes apply to? (Required)

v1.4.0.beta.3

- [x] master (the latest development version)
- [x] v1.4 (TiDB Operator 1.4 versions)
- [ ] v1.3 (TiDB Operator 1.3 versions)
- [ ] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

**Comment by IANTHEREAL** [comment]
@3pointer PTAL

File: zh/grant-permissions-to-remote-storage.md [hunk 1]
```diff
@@ -122,6 +124,14 @@ kubectl create secret generic s3-secret --from-literal=access_key=xxx --from-lit
     kubectl annotate sa tidb-backup-manager -n eks.amazonaws.com/role-arn=arn:aws:iam::123456789012:role/user --namespace=test1
     ```
 
+    如果是进行基于 AWS EBS 快照的备份和恢复，需要绑定 IAM 到 controller 的 ServiceAccount 上：
```
Comment (shichun-0415) [inline zh/grant-permissions-to-remote-storage.md:127]:
```suggestion
    如果是进行基于 AWS EBS 快照的备份和恢复，需要绑定 IAM 到 tidb-controller-manager 的 ServiceAccount 上：
```

File: zh/grant-permissions-to-remote-storage.md [hunk 2]
```diff
@@ -122,6 +124,14 @@ kubectl create secret generic s3-secret --from-literal=access_key=xxx --from-lit
     kubectl annotate sa tidb-backup-manager -n eks.amazonaws.com/role-arn=arn:aws:iam::123456789012:role/user --namespace=test1
     ```
 
+    如果是进行基于 AWS EBS 快照的备份和恢复，需要绑定 IAM 到 tidb-controller-manager 的 ServiceAccount 上：
+
+      {{< copyable "shell-regular" >}}
```
Comment (ran-huang) [inline zh/grant-permissions-to-remote-storage.md:129]:
```suggestion
```

File: en/grant-permissions-to-remote-storage.md [hunk 3]
```diff
@@ -116,14 +116,24 @@ When you use this method to grant permissions, you can [create the EKS cluster](
...
 
     {{< copyable "shell-regular" >}}
 
     ```shell
     kubectl annotate sa tidb-backup-manager -n eks.amazonaws.com/role-arn=arn:aws:iam::123456789012:role/user --namespace=test1
     ```
 
+    When backing up or restoring a TiDB cluster using EBS volume snapshots, you need to associate the IAM role with the `ServiceAccount` resources of tidb-controller-manager.
+
+      {{< copyable "shell-regular" >}}
```
Comment (ran-huang) [inline en/grant-permissions-to-remote-storage.md:131]:
```suggestion
```

**Review by ran-huang — APPROVED** [review]

**Pull request merged**